src/encoded/glossary.py

```diff
@@ -18,3 +19,17 @@ def glossary(context, request):
         'filters': []
     }
     return result
+
+
+@view_config(route_name='series_search', request_method='GET', permission='search')
+def glossary(context, request):
```

Comment:
I’m not sure of the connection /series-search/ has with glossary. Is there one?

Reply:
Updated.